x-pack/filebeat/module/fortinet/firewall/test/fortinet.log-expected.json

```diff
@@ -0,0 +1,1524 @@
+        "source.user.name": "elasticuser",
+        "tags": [
+            "fortinet-firewall"
+        ],
+        "url.domain": "elastichostname",
+        "url.path": "https://example.co/config/"
+    },
+    {
+        "@timestamp": "2020-04-23T12:17:45.000-05:00",
+        "destination.as.number": 15169,
+        "destination.as.organization.name": "Google LLC",
+        "destination.bytes": "6812",
+        "destination.geo.continent_name": "North America",
+        "destination.geo.country_iso_code": "US",
+        "destination.geo.location.lat": 37.751,
+        "destination.geo.location.lon": -97.822,
+        "destination.ip": "8.8.8.8",
+        "destination.port": "443",
+        "event.action": "ftgd_allow",
+        "event.category": [
+            "network"
+        ],
+        "event.code": "0317013312",
+        "event.dataset": "fortinet.firewall",
+        "event.kind": "event",
+        "event.module": "fortinet",
+        "event.outcome": "success",
+        "event.start": "2020-04-18T12:17:46.314-05:00",
+        "event.timezone": "-0500",
+        "event.type": [
+            "allowed"
+        ],
+        "fileset.name": "firewall",
+        "fortinet.firewall.action": "passthrough",
+        "fortinet.firewall.authserver": "elasticauth",
+        "fortinet.firewall.cat": "23",
+        "fortinet.firewall.dstintfrole": "wan",
+        "fortinet.firewall.method": "domain",
+        "fortinet.firewall.reqtype": "direct",
+        "fortinet.firewall.sessionid": "543234",
+        "fortinet.firewall.srcintfrole": "lan",
+        "fortinet.firewall.subtype": "webfilter",
+        "fortinet.firewall.type": "utm",
+        "fortinet.firewall.vd": "root",
+        "input.type": "log",
+        "log.level": "notice",
+        "log.offset": 730,
+        "message": "URL belongs to an allowed category in policy",
+        "network.bytes": "35456812",
+        "network.direction": "outgoing",
+        "network.iana_number": "6",
+        "network.protocol": "https",
+        "observer.egress.interface.name": "wan1",
+        "observer.ingress.interface.name": "port1",
+        "observer.name": "testswitch1",
+        "observer.product": "Fortigate",
+        "observer.serial_number": "somerouterid",
+        "observer.type": "firewall",
+        "observer.vendor": "Fortinet",
+        "related.ip": [
+            "192.168.2.1",
+            "8.8.8.8"
+        ],
+        "related.user": [
+            "elasticuser"
+        ],
+        "rule.category": "Web-based Email",
+        "rule.id": "38",
+        "rule.ruleset": "elasticruleset",
+        "service.type": "fortinet",
+        "source.bytes": "3545",
+        "source.ip": "192.168.2.1",
+        "source.port": "65236",
+        "source.user.group.name": "elasticgroup",
+        "source.user.name": "elasticuser",
+        "tags": [
+            "fortinet-firewall"
+        ],
+        "url.domain": "elastic.co",
+        "url.path": "https://example.com/"
```

Comment:
What is currently in url.path I would expect to be in url.full or url.original since it looks like a full url.  For url.path I would expect "/" with a url.full of "https://example.com/".  Also it still looks like the url.domain and what is the full url don't match.  For a url.full of "https://example.com/" I would expect url.domain to be "example.com"

Reply:
This was a typo in my dataset and not the mapping, fixing it now